staging/src/k8s.io/apiserver/pkg/storage/tests/cacher_test.go

```diff
@@ -443,9 +443,8 @@ func TestWatch(t *testing.T) {
 		t.Fatalf("Expected no direct error, got %v", err)
 	}
 	defer tooOldWatcher.Stop()
-	// Ensure we get a "Gone" error
-	expectedResourceExpiredError := errors.NewResourceExpired("").ErrStatus
-	verifyWatchEvent(t, tooOldWatcher, watch.Error, &expectedResourceExpiredError)
+	// Events happens in eventFreshDuration, cache expand without event "Gone".
+	verifyWatchEvent(t, tooOldWatcher, watch.Added, podFoo)
```

Comment:
Changing this one is probably fine, as it's more of an integration test from the watch_cache perspective.

Reply:
Reserved this change because `defaultLowerBoundCapacity` is a private member.